src/Core/TileGeometry.js

```diff
@@ -31,6 +31,10 @@ class TileGeometry extends THREE.BufferGeometry {
 
         this.computeBoundingBox();
         this.OBB = {};
+        this.hideSkirt = params.hideSkirt;
```

Comment:
Not really satisfied about that. The setter will be called everytime, maybe : 
```suggestion
        if(params.hideSkirt){
           this.hideSkirt = params.hideSkirt;
           }
```
OR

```suggestion
        if(params.hideSkirt){
           this.setDrawRange(0, getBufferIndexSize(this.segments, params.hideSkirt));
           }
```

WDYT @Desplandis  ?

Reply:
I agree, we should never have a property and a setter of the same name. I prefer the second solution since we do not duplicate code.